modules/ROOT/pages/delete-endpoints.adoc

```diff
@@ -1,22 +1,17 @@
 = Delete Endpoints
 
-You can delete endpoints that are not used in message flow configurations from the Host or Partner page:
+You can delete endpoints that are not used in message flows from the Host or Partner page:
```

Comment:
```suggestion
If they are not being used, you can delete endpoints from the Host or Partner page:
```
NOTE that "Host or Partner page" are capitalized here and not at Line 3, and that, again, it is unclear whether this means...:

Either the host page or the partner page
Either the host (not a page) or the partner page